packages/flutter/lib/src/material/drawer_theme.dart

```diff
@@ -0,0 +1,158 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ui' show lerpDouble;
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/rendering.dart';
+import 'package:flutter/widgets.dart';
+
+import 'theme.dart';
+
+/// Defines default property values for descendant [Drawer] widgets.
+///
+/// Descendant widgets obtain the current [DrawerThemeData] object
+/// using `DrawerTheme.of(context)`. Instances of [DrawerThemeData] can be
```

Comment:
The API doc doesn't appear to include a link to DrawerTheme.  It should at least be in the See Also section

Reply:
Added